Tests/Tests.swift

```diff
@@ -59,6 +59,11 @@ class Tests: XCTestCase {
         expected = "😍"
         XCTAssertEqual(result, expected)
 
+        string = "👯try longer😍"
+        result = string[11]
+        expected = "😍"
+        XCTAssertEqual(result, expected)
```

Comment:
Checking that even if you mix and match emoji and regular text that the indices are still correct. Actually this test is incomplete. Adding another.